test/extended/apiserverauth/api.go

```diff
@@ -0,0 +1,57 @@
+package apiserverauth
+
+import (
+	"strings"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	"github.com/tidwall/gjson"
+
+	exutil "github.com/kcp-dev/kcp-tests/test/extended/util"
+)
+
+var _ = g.Describe("[sig-API_Server]", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		k = exutil.NewCLIWithoutNamespace("kcp-api")
+	)
+
+	g.It("Author:knarra-Medium-1001-[Smoke] Checking apiserver version should display correctly", func() {
+		g.By("# Check the apiserver version display correctly")
+		out, err := k.Run("version").Args("-o", "json").Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		kcpServerVersion := gjson.Get(out, `serverVersion.gitVersion`)
+		o.Expect(kcpServerVersion).Should(o.ContainSubstring("kcp-v"))
+		if strings.Contains(k.WorkSpace().ServerURL, "kcp-stable.apps") {
+			o.Expect(kcpServerVersion).ShouldNot(o.ContainSubstring("v0.0.0"))
+		}
+	})
+
+	g.It("Author:knarra-Medium-1002-[Smoke] Multi levels workspaces lifecycle should works well", func() {
+		g.By("# Create a test workspace should become ready to use")
+		k.SetupWorkSpace()
```

Comment:
@Phaow i understand the difference between SetupWorkspace and CreateWorkSpace is one is being created in the usershome  and other one at the org level, please let me know if this is incorrect.

Reply:
`SetupWorkspace` is being created usershome, `CreateWorkSpace` will use the current workspace to create the child  workspace.